commons-httpclient/src/main/java/org/apache/commons/httpclient/HttpMethodDirector.java

```diff
@@ -150,9 +150,10 @@ public void executeMethod(final HttpMethod method) throws IOException, HttpExcep
         
                 // get a connection, if we need one
                 if (this.conn == null) {
+                    connectionManager.closeIdleConnections(60000);
                     this.conn = connectionManager.getConnectionWithTimeout(
                         hostConfiguration,
-                        this.params.getConnectionManagerTimeout() 
+                        this.connectionManager.getParams().getConnectionTimeout()
```

Comment:
this.params.getConnectionManagerTimeout return hardcoded value.

It is hardcoded at org.apache.axis2.transport.http.AbstractHTTPSender.
 - https://github.com/wso2/wso2-axis2/blob/master/modules/transport/http/src/org/apache/axis2/transport/http/AbstractHTTPSender.java#L647